### PR TITLE
Fix : Throw a better message if http-client is not installed

### PR DESCRIPTION
**Pull request by Zowac**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | Yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #2982 
| License       | MIT
| Doc PR        | api-platform/doc#1234

### CODE REVIEW

File: src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php [hunk 1]
```diff
@@ -129,6 +130,10 @@ public function load(array $configs, ContainerBuilder $container): void
             ->addTag('api_platform.filter');
 
         if ($container->hasParameter('test.client.parameters') && class_exists(AbstractBrowser::class)) {
+            if (!class_exists(HttpClient::class)){
+                throw new RuntimeException('Run "composer require symfony/http-client" to use the test client"');
```
Comment (dunglas) [inline src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php]:
```suggestion
                throw new RuntimeException('Run "composer require --dev symfony/http-client" to use the test client"');
```

Can you also rebase and run PHP CS Fixer please?

Comment (Zowac) [reply src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php]: Done :)

**Comment by dunglas** [comment]
Thanks @Zowac!

**Comment by dunglas** [comment]
Actually, I've probably be too fast to merge. It would be better to throw here: https://github.com/api-platform/core/blob/master/src/Bridge/Symfony/Bundle/Test/ApiTestCase.php#L44

Currently the exception will be triggered for every Symfony application not having the HttpClient installed, even if it doesn't use our test client.

**Pull request merged**